legacy/src/api/assets/tokens.md

```diff
@@ -70,3 +70,110 @@ Token value may be set in multiple currencies and is the same for all tokens in
+  })
+{% endreqspec %}
+
+{% h4 Fields %}
+
+|     Field      |  Type  |                                      Description                                       |
+| :------------- | :----- | :------------------------------------------------------------------------------------- |
+| name   | String | The display name of the collection. |
+| accountId | String | The account that will own the collection.                              |
+| tokenExpiresAfter   | String | The active duration of all tokens created from this collection. |
+| type | String | Supported values: <br> - `product`: tokens for this collection will be exchanged for one product. |
+| maxValue | {% dt Monetary %} {% opt %} | The maximum agreed value that any merchants will be settled for a token redemption. |
+| mediaUploadId | String {% opt %} | The id of the media upload image used for all tokens that are created from this collection. |
+
+{% h4 TokenExpiresAfter %}
+
+|     Field      |  Type  |                                      Description                                       |
+| :------------- | :----- | :------------------------------------------------------------------------------------- |
+| period | String | The unit of time to measure the duration until token expiration. It can be `hours`, `days`, `weeks`, and `months` .|
+| duration | String | The length of time for the token to remain valid before expiration. |
+
+{% h4 Example response payload %}
+{% json %}
+{
+  "id": "Xv990BzkgfoDS7bBls50pd",
+  "name": "Bread",
+	"accountId": "T3y6hogYA4d612BExypWYH",
+	"img": "https://static.centrapay.com/assets/media-uploads/{mediaUploadId}.png",
+  "tokenExpiresAfter": {
+    "period": "months",
+    "duration": "1",
+  },
+  "maxValue": {
+		"currency": "NZD",
+		"amount": "400",
+	},
+  "test": true,
+	"type": "product",
+	"status": "active",
+	"createdBy": "crn::user:b657195e-dc2f-11ea-8566-e7710d592c99",
+	"createdAt": "2021-05-12T04:30:11.001Z",
+}
+{% endjson %}  
+
+## Create Redemption Condition
+
+{% reqspec %}
+  POST '/api/collections/{collectionId}/redemption-conditions'
+  auth 'api-key'
+  path_param 'collectionId', 'NFhUgPQEYbk2EbTXAYArTX'
+  body ({
+   "merchantId": "36EALpZ89XpShxM2Ee9sXT",
+		"allowedProducts": [
+			{ "sku": "100001", "name": "White Bread", "maxValue": { "currency": "NZD", "amount": "400" },}, 
+			{ "sku": "100002", "name": "Sourdough Bread", "maxValue": { "currency": "NZD", "amount": "800" },}, 
+		]
+  })
+{% endreqspec %}
+
+{% h4 Fields %}
+
+|     Field      |  Type  |                                      Description                                       |
+| :------------- | :----- | :------------------------------------------------------------------------------------- |
+| merchantId   | String | The ID of the merchant that is accepting the collection. |
+| allowedProducts | String {% opt %} | List of allowed products, required for collections of type `product`                             |
+
+{% h4 AllowedProducts %}
```

Comment:
I don't think these should be h4, maybe ### allowedProducts? Example is Params in the createInvitation API

Reply:
Yep :)  Thanks for the example 👍